src/utils/stringify.ts

```diff
@@ -1,3 +1,7 @@
-export default function stringify(data: string) {
+export function stringify(data: string) {
 	return JSON.stringify(data.replace(/([^\\@#])?([@#])/g, '$1\\$2'));
```

Comment:
`stringify` should probably call `escape` instead of having its own copy of the regex.